core/blockchain.go

```diff
@@ -373,7 +382,7 @@ func (bc *BlockChain) getHeaderModeTBCEVMHeader() (*types.Header, error) {
 	if header != nil {
 		return header, nil
 	}
-	return nil, fmt.Errorf(fmt.Sprintf("Unable to find EVM header corresponding to hash %x", stateBlockHash[:]))
+	return nil, fmt.Errorf("Unable to find EVM header corresponding to hash %x", stateBlockHash[:])
```

Comment:
Go errors should start with a lowercase character:
```suggestion
	return nil, fmt.Errorf("unable to find EVM header corresponding to hash %x", stateBlockHash[:])
```

Reply:
Thanks 👍